quirks/websites-with-shared-credential-backends.json

```diff
@@ -240,6 +240,12 @@
         "lookmark.link"
     ],
     [
+        "claro.com.br",
+        "clarogaming.gg",
+        "netcombo.com.br",
+        "net.com.br"
+     ],
+     [
```

Comment:
My eyes might be playing tricks on me, but it looks like there are alignment issues here.

Reply:
omg. You were right. I need new glasses. Maybe from Apple? 🤔 😛